app/views/candidate_mailer/_wait_or_respond.text.erb

```diff
@@ -1,7 +1,11 @@
-^You can wait to hear back about your other application(s) before making a decision. When the last decision is in, you’ll have 10 working days to respond.
+You do not need to accept or decline this offer yet.
 
-^Or
+You can wait until you’ve received decisions about your other application(s). 
 
-^You can respond to the offer now. If you accept the offer, your other teacher training application(s) will be withdrawn. Sign in to your account if you want to respond:
+When the last decision is in, you’ll have 10 working days to respond.
+
+You should not feel pressured to respond sooner than this.
+
+However, if you are ready to respond to your existing offer you can do so through your account:
```

Comment:
Is "existing offer" slightly odd here, as you’ve only just received it? (If I understand the logic correctly)

I wonder if it's also worth spelling out what "respond" means?

```suggestion
However, if you are ready to respond, you can accept or reject offers through your account:
```

(Feel free to ignore this suggestion though!)

Reply:
Thanks Frankie. Will leave this though, as I've used this exact sentence elsewhere.